src/messagetypes/message.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from messagetypes import MsgBase
+from pybitmessage.messagetypes import MsgBase
```

Comment:
This needs to be tested outside of installed environment.

Reply:
Unit test works, but could use more testing.

Reply:
Tested & working good.